js/src/edit-block/components/top-notice.js

```diff
@@ -113,6 +125,26 @@ const TopNotice = ( { editorMode, isOnboarding } ) => {
 				</div>
 				: null
 			}
+			{
+				TEMPLATE_EDITOR_EDITING_MODE === editorMode && hasRepeaterField( getFields() )
+					? (
+						<div className="p-5 bg-blue-100 text-blue-700 border-l-4 border-blue-700 rounded-sm">
+							<p className="text-sm ml-2">
+								{ __( 'There is a repeater field, which will only display with', 'genesis-custom-blocks' ) }
```

Comment:
There's a notice to use PHP templates when using the Pro repeater field:

<img width="737" alt="repeaters-ib" src="https://user-images.githubusercontent.com/4063887/118701797-bf49c080-b7d9-11eb-8d19-ef269a6a37d0.png">

That field requires iteration and PHP functions.